### PR TITLE
HW2_Chongzhao Mao_141

### PR DESCRIPTION
**Pull request by kuonangzhe**

### CODE REVIEW

**Comment by kuonangzhe** [comment]
HW2, two .sol files

File: Lesson-2/assignment/hw2_p2.sol [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        
+        totalSalary -= employee.salary;
+    }
+    
+    function updateEmployee(address employeeId, uint salary) {
+        require(msg.sender == owner);
+        var (employee, index) = _findEmployee(employeeId);
+        assert(employee.id != 0x0);
+        
+        totalSalary = totalSalary + salary * 1 ether - employees[index].salary * 1 ether;
```
Comment (brightq) [inline Lesson-2/assignment/hw2_p2.sol:63]: 这里可以做个`newsalary = salary * 1 ether` 然后63 行和66行不用计算两遍 能省gas。

**Review by brightq — APPROVED** [review]
非常好，Approved

**Comment by kuonangzhe** [comment]
@brightq Just modified as you suggested and sent a PR again.  Thanks for the help!

**Pull request merged**